src/update_flash.c

```diff
@@ -346,14 +431,9 @@ static int wolfBoot_delta_update(struct wolfBoot_image *boot,
         wb_flash_erase(boot, sector * WOLFBOOT_SECTOR_SIZE, WOLFBOOT_SECTOR_SIZE);
         sector++;
     }
-    /* mark that our sector flags aren't reliable in testing mode */
-    st = IMG_STATE_FINAL_FLAGS;
-    wolfBoot_set_partition_state(PART_UPDATE, st);
-    /* mark boot partition as testing */
-    st = IMG_STATE_TESTING;
-    wolfBoot_set_partition_state(PART_BOOT, st);
+    /* start re-entrant final erase */
+    wolfBoot_swap_and_final_erase(0);
```

Comment:
Why is the return code not checked?

Reply:
return code is only to tell wolfBoot_start if the final swap was resumed or not

Reply:
Then please document this in the code at this line.... why we don't need to check the return code.